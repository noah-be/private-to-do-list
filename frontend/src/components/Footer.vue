<template>
  <footer>
    <p>Made with love in Berlin</p>
  </footer>
</template>

<style lang="scss">
// footer {
//   margin-top: auto;
//   background-color: $primary-color;
//   color: $secondary-color;
//   display: flex;
//   justify-content: space-between;
// }
</style>
