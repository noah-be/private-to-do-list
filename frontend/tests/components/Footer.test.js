import { describe, it, expect } from "vitest";
import { mount } from "@vue/test-utils";
import Footer from "@/components/Footer.vue";

describe("Footer.vue", () => {
  it("renders correctly and matches snapshot", () => {
    const wrapper = mount(Footer);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
