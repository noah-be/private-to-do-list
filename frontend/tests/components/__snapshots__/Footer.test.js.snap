// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Footer.vue > renders correctly and matches snapshot 1`] = `
"<footer>
  <p>Made with love in Berlin</p>
</footer>"
`;
